src/test/java/qna/domain/DeleteHistoriesTest.java

```diff
@@ -0,0 +1,55 @@
+package qna.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class DeleteHistoriesTest {
+
+    private Question question;
+    private Answer answer;
+    private DeleteHistories deleteHistories;
+
+    @BeforeEach
+    public void setUp() {
+        question = new Question(1L, "title1", "contents1").writeBy(UserTest.JAVAJIGI);
+        answer = new Answer(1L, UserTest.JAVAJIGI, question, "Answers Contents1");
+        question.addAnswer(answer);
+
+        deleteHistories = new DeleteHistories();
+        deleteHistories.add(DeleteHistory.ofQuestion(question.getId(), question.writer(), LocalDateTime.now()));
+    }
+
+    @DisplayName("DeleteHistories에 DeleteHistory를 추가한다.")
+    @Test
+    void add() {
+        //given & when
+        deleteHistories.add(DeleteHistory.ofAnswer(answer.getId(), answer.writer(), LocalDateTime.now()));
```

Comment:
테스트 코드를 간단하게 유지하기 위해서 최소한의 코드로 구현해본다면 이렇게 할수도 있을것같아요!
이렇게 된다면 `answer`,`question`을 굳이 생성안해도 되지 않을까요?
```suggestion
        deleteHistories.add(DeleteHistory.ofAnswer(1L, UserTest.JAVAJIGI, LocalDateTime.now()));
```

Reply:
말씀하신대로 테스트 케이스를 수정하니 소스가 정말 심플해졌습니다. 👍
수정하겠습니다 😄